ISM43362/ISM43362.h

```diff
@@ -248,6 +250,29 @@ class ISM43362 {
     // Connection state reporting
     nsapi_connection_status_t _conn_status;
     mbed::Callback<void()> _conn_stat_cb;
+
+    string CountryCodeElevenChannels[12] = {"AS", "CA", "FM", "GU", "KY", "MP", "PR", "TW",
```

Comment:
Have you checked that those string tables are well lying in ROM not in RAM  ?
Also I am not sure about the point of those tables as the function check_country_code seems to accept only 4 possible country codes.

Reply:
| Module                            |           .text |       .data |          .bss |
|-----------------------------------|-----------------|-------------|---------------|
wifi-ism43362\ISM43362            |     6170(+6170) |       0(+0) |         2(+2) |

Total Static RAM memory (data + bss)

So, I am thinking these tables are not in RAM.

Explanation concerning usefulness of these tables are present in comment of function check_country_code()